pages/home/changelog.mdx

```diff
@@ -7,6 +7,31 @@ description: "What's new at Arcade.dev"
 
 *Here's what's new at Arcade.dev!*
 
+
+## For the week ending on 2025-08-15
+
+<Note>
+  This week we enforced a new requirement for all OAuth providers: they must have a unique callback URL.  This is a minor security change, but does require you to update your OAuth configuration.  This can be updated from the dashboard.
+</Note>
+
+**Toolkits**
+* `[feature - 🚀]` Sharepoint Toolkit added ([PR #30](https://github.com/ArcadeAI/toolkits/pull/30))
```

Comment:
We can point to the toolkit docs instead: https://docs.arcade.dev/toolkits/productivity/sharepoint